lib/Completion/Bridge/TolerantParser/CompletionContext.php

```diff
@@ -51,6 +52,7 @@ public static function expression(?Node $node): bool
             $parent instanceof Expression ||
             $parent instanceof StatementNode ||
             $parent instanceof ConstElement ||
+            $parent instanceof Attribute ||
```

Comment:
it's not valid to have an expression in an attribute though... !?

Reply:
Maybe it's a hack.  Would that mean that it should be checked separately, especially in some new Completor mentioned in #1999?

Reply:
yes, probably there should be a `attributeClass(Node $node)` method here - but equally can be done in the new `Completor`